tests/testthat/test-08-seasons.R

```diff
@@ -8,10 +8,10 @@ test_weather = weather("monthly",path=".") %>% dplyr::filter(year %in% 1989:2010
 test_that("add_seasons returns expected results", {
 
   rodent_seasons = add_seasons(test_rodents,level = "treatment", date_column = "period",season_level=2, path = ".")
-  expect_true(expect_true(sum(rodent_seasons$abundance, na.rm = T) == 16760))
+  expect_true(expect_true(sum(rodent_seasons$abundance, na.rm = T) == 16452))
```

Comment:
If you're ok with the default output changing, that's fine. Just curious if you know what is causing it

Reply:
Ya. Basically, the old `find_incomplete_censuses` determined what was 'complete' nominally (a census was complete if the maximum possible effort was done). Now it's based on actual effort, even if more effort wasn't possible at the time.

Reply:
word!